Application/EdFi.Ods.Api/Authorization/PoliciesAuthorizationHandler.cs

```diff
@@ -0,0 +1,47 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+#if NETCOREAPP
+using Microsoft.AspNetCore.Authorization;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace EdFi.Ods.Api.Authorization
+{
+    public class PoliciesAuthorizationHandler : AuthorizationHandler<CustomUserRequireClaim>
```

Comment:
To add to what Jim mentioned, all this controller requires is verifying the existence of a claim.  This is simple enough that I probably wouldn't recommend doing a custom AuthorizationHandler and Requirement, and instead configure it programattically using the UseAuthorization entry point on the services collection.  This is a good indication that we probably want some mechanism to let features impact Authorization configuration, but for now its probably fine to have a feature check and setup in the main Startup class.  See https://docs.microsoft.com/en-us/aspnet/core/security/authorization/policies?view=aspnetcore-3.1#use-a-func-to-fulfill-a-policy for details, but basically you need this code in Startup (with constants set up for the policy names and claim names involved).  Then you can use the policy name in your Authorize attribute on the controller:
```csharp
services.AddAuthorization(options =>
{
     options.AddPolicy("IdentityManagement", policy =>
        policy.RequireAssertion(context =>
            context.User.HasClaim(c => c.Type == "http://ed-fi.org/ods/identity/claims/domains/identity"));
});
```

Reply:
Or use the helper UserRequireCustomClaim works too, but the main thing is the check is fully contained in the AddAuthorization section, not mixing approaches.